### ISSUE TITLE
Background

### ISSUE THREAD

**Issue by astush1001** [astush1001/Calculate-Me#1]
Add a Gradient or other eye catching background.

**Comment by Soumya624** [comment]
Hi @astush1001 ,

Started working on this. Kindly assign!

**Comment by suresh8919** [comment]
Hey @astush1001 I'm about to done this will u assign me this?

**Comment by astush1001** [comment]
Yeah @Soumya624 and @suresh8919 , you guys can contribute with your suggestions and changes.
I'm assigning this issue to both of you.

**Comment by okprateek** [comment]
Hello, @astush1001 will you assign me this issue ? I have a good knowledge of UI/UX.

**Comment by anushkapal28** [comment]
Hello! I'm interested in working on this task. I have experience with HTML and CSS , and I'm eager to contribute. Please consider assigning this to me, and I'll get started right away. Thanks! Please assign me too.
